public/static/locales/de/profile.json

```diff
@@ -7,14 +7,19 @@
     "feature": {
       "redeem": "Einlösen",
       "registerTree": "Baum registrieren",
-      "share": "Teilen"
+      "share": "Teilen",
+      "allDonations": "Alle Spenden",
+      "supportUserTextIndividual": "Unterstützen Sie {username} mit einer Spende",
+      "supportUserTextGeneric": "Unterstütze sie mit einer Spende",
+      "website": "Meine Website"
     },
     "myPublicProfile": {
       "support": "Support",
       "link": "Link"
     },
     "myForestMap": {
       "plantedTree": "{count, plural, =1 {# Baum} other {# Bäume}}",
+      "savedArea": "{area} m²",
```

Comment:
"tpoName": "By {tpo}" is missing a translation below!

Reply:
@mariahosfeld small error from missing translation:
<img width="840" alt="Bildschirmfoto 2024-07-10 um 12 57 09" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/1532418/12ec6fdf-3a5f-4e1c-97ae-2efd0e70cee1">